python/hail/linalg/blockmatrix.py

```diff
@@ -386,9 +403,9 @@ def write_from_entry_expr(entry_expr, path, block_size=None):
         if not block_size:
             block_size = BlockMatrix.default_block_size()
 
-        check_entry_indexed('write_from_entry_expr/entry_expr', entry_expr)
+        check_entry_indexed('BlockMatrix.write_from_entry_expr/entry_expr', entry_expr)
```

Comment:
Why do you have the extra `/entry_expr`?

Reply:
it's tim's notation, method followed by bad parameter

Reply:
usually I only do this if there are multiple expressions to disambiguate. seems ok though, I don't really care